frontend/src/App.js

```diff
@@ -20,7 +20,7 @@ function App() {
     <div>
       <BrowserRouter>
         <div>
-          <div className = 'landing'>
+          <div key={Date.now()} className = 'landing' style={window.location.pathname === '/' ? {height: "100%"} : {}}>
```

Comment:
Why are we doing this? I deleted the key and styling + line 8-10 in index.css and the background works as intended

Reply:
That was residual code from before that bug fix, will remove asap

Reply:
I meant the inline style too. When you delete line 8-10 in index.css and remove the inline styling, everything works. Do you see different on your computer?

Reply:
Oh I'll get on removing that too, that was also residual